test/int/adapter.test.ts

```diff
@@ -85,5 +89,52 @@ suite('Chrome Debug Adapter etc', function () {
                 dc.assertStoppedLocation('debugger_statement', { path: breakFile, line: DEBUGGER_LINE } )
             ]);
         });
+
+        test('Should attach to existing instance of chrome and break on debugger statement', async () => {
+            const testProjectRoot = path.join(DATA_ROOT, 'intervalDebugger');
+            const breakFile = path.join(testProjectRoot, 'src/app.ts');
+            const DEBUGGER_LINE = 2;
+
+            server = createServer({ root: testProjectRoot });
+            server.listen(7890);
+            const browser = await puppeteer.launch(<puppeteer.LaunchOptions>{ headless: false, args: ['http://localhost:7890', '--remote-debugging-port=7777'] });
```

Comment:
Why is this cast needed?

Reply:
Not needed for anything except intellisense. For whatever reason, VS Code doesn't provide intellisense on anonymous objects, even when the type of the argument is known for that function unless you provide a type assertion. We can remove it if you'd prefer, I typically just toss them on there when I need to check what the possible values can be.